kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -626,6 +626,14 @@ object KyuubiConf {
       .intConf
       .createWithDefault(1000000)
 
+  val ENGINE_SPARK_MAX_ROWS: ConfigEntry[Int] =
+    buildConf("session.engine.spark.max.rows")
```

Comment:
kyuubi.operation.max.result.rows?

Reply:
I saw that there is a parameter named `kyuubi.session.engine.flink.max.rows`.

Reply:
How about fall back to kyuubi.operation.max.result.rows if kyuubi.session.engine.*.max.rows is not specified?

Reply:
it is to align with `kyuubi.session.engine.flink.max.rows`

Reply:
`kyuubi.operation.max.result.rows` makes sense to me, and we can deprecate `kyuubi.session.engine.flink.max.rows`

Reply:
we can deprecate kyuubi.session.engine.flink.max.rows in another pr.

Reply:
resolved